### PR TITLE
Allow to select role when sigin in debug mode

### PR DESCRIPTION
**Pull request by notmd**

### CODE REVIEW

File: website/src/lib/auth.ts [hunk 1]
```diff
@@ -1,11 +1,15 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 import { getToken, JWT } from "next-auth/jwt";
+import { ElementOf } from "src/types/utils";
 
+export const roles = ["general", "admin", "banned"] as const;
+
+export type Role = ElementOf<typeof roles>;
```
Comment (fozziethebeat) [inline website/src/lib/auth.ts:7]: Could this be put somewhere under `src/types`? That way the react components don't need to import stuff from `src/lib/auth.ts`

**Review by fozziethebeat — CHANGES_REQUESTED** [review]

File: website/src/lib/auth.ts (hunk 1 above)
Comment (notmd) [reply website/src/lib/auth.ts:7]: How about put it in the `RoleSelect` file? The `roles` const is not being used anywhere except the `RoleSelect`. About the `Role` type, Typescript will remove the import when we use the `import type`  syntax, so we don't have to worry about tree shaking here.

Comment (fozziethebeat) [reply website/src/lib/auth.ts:7]: Sure that works.

Comment (notmd) [reply website/src/lib/auth.ts]: Done!

**Review by fozziethebeat — APPROVED** [review]

**Pull request merged**